webapp/src/utils.ts

```diff
@@ -376,3 +378,50 @@ export function sendDesktopEvent(event: string, data?: Record<string, unknown>)
 export function capitalize(input: string) {
     return input.charAt(0).toUpperCase() + input.slice(1);
 }
+
+export async function fetchTranslationsFile(locale: string) {
+    if (locale === 'en') {
+        return {};
+    }
+    try {
+        // eslint-disable-next-line global-require
+        const filename = require(`../i18n/${locale}.json`).default;
```

Comment:
Could you explain what's going on here? I don't think I get it

Reply:
The require is purely needed to figure out the actual filename for the translation asset to be then fetched asynchronously (HTTP fetch call). Since the webpack generated filenames include the hash we cannot simply use the locale string to figure it out.